components/inputs/input-time.js

```diff
@@ -69,6 +86,16 @@ class InputTime extends LitElement {
 				label {
 					display: block;
 				}
+				.d2l-input-time-menu {
+					border-top: 1px solid var(--d2l-color-gypsum);
+					border-bottom: 1px solid var(--d2l-color-gypsum);
```

Comment:
Hmm... ideally we're not setting borders and stuff on things like menus -- since they already have their own borders and us "knowing about them" makes the code a lot more fragile. Why is this needed?

Reply:
After checking - the bottom isn't helping, I can get rid of it, that border can come from the header and footer slots now that they exist. The top is the difference between these two: 
![image](https://user-images.githubusercontent.com/50635849/76892256-c89b6e00-6860-11ea-92df-8b69df1e560b.png) that horizontal line under the top 'padding' isn't there (without this line) unless we have header content or you've scrolled a bit, so you kind of get what looks like inconsistent whitespace and a hovering scrollbar. If we're fine with that I can get rid of both of them.

Reply:
If a consistent border is something we want, I'd expect us to fix this at the dropdown-menu level, not in this instance of it. @geurts do you have opinions here? Dropdowns seem to only spawn that top border when there's a header or if they're scrolled -- are you OK with time inputs just behaving the same way?

Reply:
Minor input: most menus don't have a footer by default so there isn't a concern with them being 'bottom heavy' or vertically asymmetrical. In this case we have the timezone in the footer at the bottom so not having a visible top 'cap' isn't necessarily balanced. But that's an opinion, I'll defer to Jeff's judgement. :)

Reply:
If that's the logic we'd like to apply, I think we could still put it inside dropdown/menu where the header/footer is defined. They know that there's a footer but not a header so could apply the border if that's how we want to treat it.